code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,1135 @@
+	if(!locate(/obj/effect/hotspot) in rune_turf) //will create a hotspot to burn items passing through too.
+		new /obj/effect/hotspot(rune_turf)
+
+	for(var/mob/living/on_fire in rune_turf)
+		on_fire.adjust_fire_stacks(20)
+		on_fire.IgniteMob()
+
+
+/*The purple warp rune makes suture and ointment if you put cloth or plastic on it. */
+
+
+/obj/item/slimecross/warping/purple
+	colour = "purple"
+	runepath = /obj/effect/warped_rune/purplespace
+	effect_desc = "Draws a rune that transforms plastic into regenerative mesh and cloth into suture"
+
+
+/obj/effect/warped_rune/purplespace
+	desc = "When all that was left were plastic walls and the clothes on their back, they knew what they had to do."
+	icon_state = "purplespace"
+	max_cooldown = 30
+	///object path of the suture spawned
+	var/obj/item/stack/medical/suture/suture
+	///object path of the regenerative mesh spawned
+	var/obj/item/stack/medical/mesh/regen_mesh
+
+/obj/effect/warped_rune/purplespace/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+
+/obj/effect/warped_rune/purplespace/process()
+	if(cooldown < world.time)
+		cooldown = world.time + max_cooldown
+		transmute_heal()
+
+
+///transforms cloth and plastic into suture and regenerative mesh
+/obj/effect/warped_rune/purplespace/proc/transmute_heal()
+	for(var/obj/item/stack/sheet/plastic/plastic in rune_turf)  //replace plastic with regenerative mesh
+		if(plastic.amount < 2)
+			return
+
+		plastic.amount -= 2
+		regen_mesh = new (rune_turf,1)
+		playsound(rune_turf, 'sound/effects/splat.ogg', 20, TRUE)
+		if(plastic.amount <= 0)
+			qdel(plastic)
+
+	for(var/obj/item/stack/sheet/cloth/cloth in rune_turf) //replace cloth with suture
+		if(cloth.amount < 2)
+			return
+
+		cloth.amount -= 2
+		suture = new(rune_turf, 1)
+		playsound(rune_turf, 'sound/effects/splat.ogg', 20, TRUE)
+		if(cloth.amount <= 0)
+			qdel(cloth)
+
+
+/* the blue warp rune  keeps a tile slippery CONSTANTLY by adding lube over it. Excellent if you hate standing up.*/
+
+
+/obj/item/slimecross/warping/blue
+	colour = "blue"
+	runepath = /obj/effect/warped_rune/cyanspace //we'll call the blue rune cyanspace to not mix it up with actual bluespace rune
+	effect_desc = "creates a rune that constantly wet itself with slippery lube as long as the rune is up"
+
+
+/obj/effect/warped_rune/cyanspace
+	icon_state = "slipperyspace"
+	desc = "You will crawl like the rest. Standing up is not an option."
+
+
+/obj/effect/warped_rune/cyanspace/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+
+/obj/effect/warped_rune/cyanspace/process()
+	slippery_rune(rune_turf)
+
+
+///Spawn lube on the tile the rune is on every process, Cannot lube a non open turf
+/obj/effect/warped_rune/cyanspace/proc/slippery_rune(turf/open/lube_turf)
+	lube_turf.MakeSlippery(TURF_WET_LUBE,min_wet_time = 10 SECONDS, wet_time_to_add = 2 SECONDS)
+
+
+/*Metal rune : makes an invisible wall. actually I lied, the rune is the wall.*/
+
+
+/obj/item/slimecross/warping/metal
```

Comment:
nope, it blocks out any projectile from passing by it, although it will act as if the rune is hit which I guess it technically is, the rune won't take any damage from it. It's basically a see through wall.